packages/core/useWebSocket/index.ts

```diff
@@ -25,12 +27,12 @@ export interface UseWebSocketOptions {
      *
      * @default 'ping'
      */
-    message?: string | ArrayBuffer | Blob
+    message?: WebSocketHeartbeatMessage | (() => WebSocketHeartbeatMessage)
```

Comment:
you should consider `MaybeRefOrGetter<WebSocketHeartbeatMessage>`

Reply:
I'm sorry. I'll check this out function😢